inbox/events/ical.py

```diff
@@ -79,9 +79,10 @@ def events_from_ics(namespace, calendar, ics_str):
 
                 if original_start.tzinfo is None:
                     tzid = component.get("dtstart").params.get("TZID", None)
-                    assert (
-                        tzid in timezones_table
-                    ), "Non-UTC timezone should be in table"
+                    if tzid not in timezones_table:
+                        raise MalformedEventError(
+                            "Non-UTC timezone should be in table", tzid
+                        )
```

Comment:
I'll eventually turn this into `log.warning` when I see that we are getting small number of occurrences that we cannot recover from.